src/components/navbar/Navbar.js

```diff
@@ -2,12 +2,17 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 
 const Navbar = () => (
-  <nav className="navbar">
+  <Navbar>
+    <h1>Bookstore</h1>
     <ul>
-      <Link className="navbar-links" to="/">Books</Link>
-      <Link className="navbar-links" to="categories">Categories</Link>
+      <li>
+        <Link to="/">Books</Link>
+      </li>
+      <li>
+        <Link to="/categories">Categories</Link>
+      </li>
     </ul>
-  </nav>
+  </Navbar>
```

Comment:
- [ ] When I run your app it does not really render anything - because it loads forever. This is because you are wrapping your Navbar component with a **`Navbar`** element which does not exist. I would suggest you wrap your navbar with **`nav`** instead of `Navbar`